app/lib/pre_assembly/batch.rb

```diff
@@ -71,20 +71,22 @@ def object_filenames_unique?(dobj)
     # For each container, creates a new Digitalobject.
     # @return [Array<DigitalObject>]
     # rubocop:disable Metrics/AbcSize
-    def digital_objects
-      @digital_objects ||= discover_containers_via_manifest.each_with_index.map do |c, i|
+    def digital_objects(&_block)
```

Comment:
what do you think of renaming this method `digital_objects_from_containers` ?   Neutral question, fine for you not to like it.

Reply:
i slightly prefer the current name, and so left it as is, but don't feel strongly